libs/editor/egui_editor/src/offset_types.rs

```diff
@@ -414,16 +414,17 @@ impl Debug for RelCharOffset {
     }
 }
 
-pub trait RangeExt<Element>: Ord + Sized {
+pub trait RangeExt<Element: Sub<Element>>: Ord + Sized {
```

Comment:
fun rust syntax for "a trait implemented for types which can be subtracted from themselves and which implies `Ord` and `Sized`"